### PR TITLE
Release 2018

### PR DESCRIPTION
**Pull request by Ryuno-Ki**
Thanks, @Enlik and darksurf for collaboration!

@mudler I think, this is ready to be published.

### CODE REVIEW

**Comment by Ryuno-Ki** [comment]
Hm, looks like the rebase went somewhat not right … I'll investigate it.

File: content/article/sabayon-is-back-and-kicking.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+The focus of the webmaster will be on the website first, then turning to the
+wiki before looking on improving the forums.
+
+If you have ideas, feel free to drop them at
+ryuno-ki at sabayonlinux.org.
+
+## New release strategy
+
+Sabayon used to release a new version every month.
```
Comment (mudler) [inline content/article/sabayon-is-back-and-kicking.md]: Please, don't confuse our users even more, we already changed release strategy, and explained it [here under "Changes"](https://www.sabayon.org/release/latest-monthly-release-sabayon-1611/) .

Automatic (not always tested) monthly release will always be there, but we use to tag them manually into [stable/ directory](http://mirror3.mirror.garr.it/mirrors/sabayonlinux/stable/) when we decide to cut a new release.

We can change the timing of cutting them, but i don't think it's a good idea to remove automatic builds.

**Review by mudler — CHANGES_REQUESTED** [review]

File: content/article/sabayon-is-back-and-kicking.md (hunk 1 above)
Comment (Ryuno-Ki) [reply content/article/sabayon-is-back-and-kicking.md]: Addressed in https://github.com/Sabayon/sabayon-website-next/pull/63/commits/bf11b1a198e7f20591d3e0ec5fb76cbe9b5e5505

**Comment by Ryuno-Ki** [comment]
@mudler Please give it another shot!

**Comment by mudler** [comment]
Looks good to me :-)

**Comment by jcfrosty** [comment]
Cool, glad we got it worked out!

**Pull request merged**